src/canvas/App.tsx

```diff
@@ -0,0 +1,18 @@
+import { Renderer } from './Renderer';
+import { Document } from './Document';
+import { PathSelection } from './PathSelection';
+
+class App {
+    renderer: Renderer;
+    document: Document;
+    pathSelection: PathSelection;
+
+    constructor () {
+        this.document = new Document();
+        this.pathSelection = new PathSelection();
+        this.renderer = new Renderer(this.document, this.pathSelection);
+    }
+}
+
+const AppModels = new App();
```

Comment:
I called it AppModels because it is where I instantiated the document model, the renderer view model and the selection model. I renamed it when exporting because there is already and App in the React code. I could rename the file to AppModel or if you think that would be better. Also, please let me know if you have any naming suggestions.

Reply:
I'm good with it, now that I understand what you mean by models. So the models are like session data units?

Reply:
maybe have AppComponent for the React component and AppModels for the collection of models for our app?

Reply:
Yeah, I was going for something close to an MVC structure.  Think I'll call it AppController.

Reply:
sounds good